### PR TITLE
DBZ-3675 Using same Antlr version for compilation and testing;

### PR DESCRIPTION
**Pull request by gunnarmorling**
Also updating Antlr4 test plug-in and making variable name consistent with others.

https://issues.redhat.com/browse/DBZ-3675

### CODE REVIEW

**Review by Naros — APPROVED** [review]
LGTM, just waiting on CI then will merge.  

There will be sibling repository failures due to the updated debezium-bom changes with the property name change from `antlr.version` to `version.antlr`, which should be resolved automatically once merged.

**Comment by gunnarmorling** [comment]
> LGTM, just waiting on CI then will merge.

Thanks!

> There will be sibling repository failures due to the updated debezium-bom
changes with the property name change from antlr.version to version.antlr

I don't think that variable is used in any of the other repos, or is it?

>

**Comment by Naros** [comment]
No, I specifically checked Cassandra since it failed early on and I didn't see it being used in its pom afaict.

**Pull request merged**
